src/enemy.ts

```diff
@@ -5,8 +5,30 @@ import { DirectionsType } from './projectile';
 export class Enemy extends Tank {
   private _directions: DirectionsType[] = ['up', 'down', 'left', 'right'];
 
+  protected _upPos: string[][] = [
```

Comment:
Maybe it's better to use '+' or "O" instead of '>','<' and 'M', however, it's optional.